### PR TITLE
Register Other Org - Sprint 31.4

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
(https://tools.hmcts.net/jira/browse/EUI-8756)

### Change description ###
To be updated

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by udaydenduluri33 — APPROVED** [review]

File: api/registerOrganisation/index.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import { NextFunction, Request, Response, Router } from 'express';
+import { RegistrationData, RegistrationRequest } from '../models/registrationData';
+import { generateS2sToken } from '../lib/s2sTokenGeneration';
+import { getConfigValue } from '../configuration';
+import { SERVICES_RD_PROFESSIONAL_API_PATH, SERVICE_S2S_PATH } from '../configuration/references';
+import { http } from '../lib/http';
```
Comment (connorpgpmcelroy) [inline api/registerOrganisation/index.ts:6]: minor issue - import ordering

File: src/organisation/containers/organisation/organisation.component.ts [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
-import { select, Store } from '@ngrx/store';
-import { Subject } from 'rxjs';
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
+import { Store, select } from '@ngrx/store';
+import { Observable, Subject } from 'rxjs';
 import { takeUntil } from 'rxjs/operators';
 import { DxAddress, OrganisationContactInformation, OrganisationDetails, PBANumberModel } from '../../../models';
+import { Regulator, RegulatorType, RegulatoryType } from '../../../register-org/models';
+import { AppConstants } from '../../../app/app.constants';
```
Comment (connorpgpmcelroy) [inline src/organisation/containers/organisation/organisation.component.ts:8]: minor issue - import ordering

File: src/organisation/utils/index.ts [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import { PBANumberModel } from 'src/models/pbaNumber.model';
```
Comment (connorpgpmcelroy) [inline src/organisation/utils/index.ts:1]: src mention

File: src/register-org/components/check-your-answers/check-your-answers.component.ts [hunk 4]
```diff
@@ -1,55 +1,104 @@
...
-import { ActivatedRoute, Router } from '@angular/router';
+import { AbstractControl, FormControl, FormGroup, ValidatorFn, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { HttpErrorResponse } from '@angular/common/http';
 import { RegisterComponent } from '../../../register-org/containers';
-import { RegistrationData } from '../../models';
+import { ORGANISATION_SERVICES } from '../../constants/register-org-constants';
+import { RegulatorType, RegulatoryType } from '../../models';
 import { RegisterOrgService } from '../../services/register-org.service';
+import { ORGANISATION_TYPES_REF_DATA } from 'src/register-org/__mocks__';
```
Comment (connorpgpmcelroy) [inline src/register-org/components/check-your-answers/check-your-answers.component.ts:9]: src and import ordering

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Very minor changes requested

**Pull request merged**